integration/test_graphql.py

```diff
@@ -89,6 +89,34 @@ def test_get_data(client):
             d_found = True
     assert a_found and d_found and len(objects) == 2
 
+def test_get_data_with_properties_default(client, people_schema):
+    """Test GraphQL's Get clause with properties default (None)."""
+    client.schema.create(people_schema)
+
+    person_id = uuid.uuid4()
+    client.data_object.create(
+        {
+            "name": "John",
+        },
+        "Person",
+    )
+
+    client.data_object.create(
+        {
+            "name": "Group-Weaviate",
+            "members": [
+                {
+                    "beacon": "weaviate://localhost/" + str(person_id),
```

Comment:
@dirkkul, I need some help regarding how can I create this reference correctly.

Reply:
First add the group without references (eg leave the members out) and then use data_object.add_reference() toa dd the reference. You can check integration/test_crud.py for examples

Reply:
Thanks, made the updates.